cmake/kokkos_tribits.cmake

```diff
@@ -178,9 +178,9 @@ ENDIF()
 ENDFUNCTION()
 
 MACRO(KOKKOS_SETUP_BUILD_ENVIRONMENT)
+ INCLUDE(${KOKKOS_SRC_PATH}/cmake/kokkos_compiler_id.cmake)
```

Comment:
Good change. I think a lot of subsequent stuff could depend on the compiler so figuring that out early is good.